apps/redi-talent-pool/src/components/organisms/jobseeker-profile-editables/EditableImportantDetails.tsx

```diff
@@ -140,7 +139,7 @@ export function EditableImportantDetails({
               </div>
             ) : null}
 
-            {profile?.postalMailingAddress ? (
+            {!hideFullAddress && profile?.postalMailingAddress ? (
```

Comment:
Can we flip this from `hide` to `show`? I think it'll be easier to reason about a boolean prop called `showFullAddress`, avoiding the `!`. It has the nice side-effect of making it clearer that our preferred default is to not show the full address.

Reply:
Ah, yes gotcha. This component does indeed have various concerns and we'd be messing with client code here.
Agreed - let's not change anything for the time being. Approving PR.